src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -813,6 +815,13 @@ protected function asDateTime($value)
         return Date::createFromFormat($format, $value);
     }
 
+    protected function asTime($value)
```

Comment:
Needs a docblock and also, for consistency, move this new method to be after the `fromDateTime()` method.

Reply:
I made the specified changes.